src/partcad/adhoc/common.py

```
@@ -0,0 +1,157 @@
+        kind: "part" or "sketch", which decides how the input is declared.
+        verb: What is being done, for the progress label and the error message.
+        options: Export parameters handed to the implementation, overriding what
+            it would otherwise default to. This is where a render's viewport
+            arrives; a conversion passes none.
+
+    Raises:
+        RuntimeError: the input could not be loaded, or the output not written.
+            The cause is in the message and not only in '__cause__', because
+            this is what the CLI prints.
+    """
+    section, object_name = KINDS[kind]
```

Ruff (0.16.2)

[warning] Unpacked variable `section` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)